hystrix/hystrix_client.go

```diff
@@ -179,7 +179,7 @@ func (hhc *Client) Do(request *http.Request) (*http.Response, error) {
 		}
 
 		err = hystrix.Do(hhc.hystrixCommandName, func() error {
-			response, err = hhc.client.Do(request)
+			resp, err := hhc.client.Do(request)
```

Comment:
Can you elaborate on this fix plz ?

Reply:
There are two possible source of errors, hystrix and net/http. If both having the same timeout value, the err will be in race, as it can override one another.